spec/factories/kudos.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :kudo do
+    title { 'Title' }
+    content { 'Content' }
+    employee_id
```

Comment:
why not `employee`? https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#associations